pkg/networkpath/traceroute/traceroute_windows.go

```diff
@@ -42,7 +42,7 @@ func (w *WindowsTraceroute) Run() (NetworkPath, error) {
 		log.Warnf("could not initialize system-probe connection: %s", err.Error())
 		return NetworkPath{}, err
 	}
-	resp, err := tu.GetTraceroute(clientID, w.cfg.DestHostname)
+	resp, err := tu.GetTraceroute(clientID, w.cfg.DestHostname, l.cfg.DestPort, l.cfg.MaxTTL, l.cfg.TimeoutMs)
```

Comment:
```suggestion
	resp, err := tu.GetTraceroute(clientID, w.cfg.DestHostname, w.cfg.DestPort, w.cfg.MaxTTL, w.cfg.TimeoutMs)
```
These should be `w` for the windows struct